sw/device/silicon_creator/lib/ownership/datatypes.h

```diff
@@ -0,0 +1,29 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_OWNERSHIP_DATATYPES_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_OWNERSHIP_DATATYPES_H_
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+typedef struct owner_key {
```

Comment:
There is also this definition. https://github.com/lowRISC/opentitan/blob/3a35e26c7a1e6e26aa38f4ae899670274f69b1d1/sw/device/silicon_creator/lib/sigverify/ecdsa_p256_key.h#L21-L28

I am ok with keeping separate definitions for Ownership transfer, but it would be nice to document the key types etc.

Reply:
You, @timothytrippel and I all have need of structs that hold ECDSA keys and signatures.  We should look to unify the structures in the near future.

Reply:
#22134